src/transactions/saga.ts

```diff
@@ -307,3 +313,58 @@ export function* transactionSaga() {
   yield* spawn(watchAddressToE164PhoneNumberUpdate)
   yield* spawn(watchPendingTransactions)
 }
+
+export function* handleTransactionReceiptReceived(
+  txId: string,
+  receipt: TransactionReceipt | CeloTxReceipt,
+  networkId: NetworkId
+) {
+  const tokensById = yield* select((state) => tokensByIdSelector(state, [networkId]))
+
+  // TODO: Receive feeCurrencyId from transaction request.
+  const feeCurrencyId = getTokenId(networkId)
```

Comment:
Yes we discussed the fact the `feeCurrency` is missing from the TX receipt with cLabs and it's by design. Things that are already in the TX itself aren't present in the receipt.

https://valora-app.slack.com/archives/CNJ7KTHQU/p1698843275835969?thread_ts=1698688855.441649&cid=CNJ7KTHQU

Do you want to address the TODO in a separate PR? My only concern is it should be done relatively close to merging this one otherwise we risk merging this and display incorrect fee to the user if it's paid with another currency.

Reply:
i agree, would be nice to address this TODO asap. i think there's no risk that the wrong fee is displayed though because further in this function, we omit the fees calculation if the network is Celo?

Reply:
Yes, I prefer to address this TODO separate, because that change will require some refactor since it's not straight forward to get the `feeCurrency`, there is some abstraction in the middle.
With the current code, the CELO fees are ignored.